app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+    if(@user.nil?)
+      flash[:danger] = t "error"
+    else
+      flash[:success] = t "welcome"
+    end
```

Comment:
```suggestion
    return if @user
        
    flash[:danger] = t "error"
    redirect_to root_path
```
ri hì

chứ bjk user ko có mà còn hiển thị show thì nó cũng crash hệ thoogn thôi

Reply:
oke chj